assets/javascripts/discourse/initializers/extend-for-salesforce.js.es6

```diff
@@ -76,6 +79,7 @@ function initializeWithApi(api) {
     });
 
     TopicStatusIcons.addObject(["has_salesforce_case", "briefcase", "case"]);
+    // eslint-disable-next-line no-undef
```

Comment:
And Discourse global usage? 😉

Reply:
I think we don't have a solution for this. We're doing the same in core too
https://github.com/discourse/discourse/blob/bf33d2cd4b27bbb63720c8691f4508e4ea0e8568/app/assets/javascripts/discourse/app/models/site.js#L218